src/components/views/elements/TagTile.js

```diff
@@ -22,17 +22,36 @@ import sdk from '../../../index';
 import dis from '../../../dispatcher';
 import { isOnlyCtrlOrCmdKeyEvent } from '../../../Keyboard';
 
+import FlairStore from '../../../stores/FlairStore';
+
 export default React.createClass({
```

Comment:
generally the comments are a bit sparse round here. It's not obvious to me what a "tag" is, or what a TagTile is. Just raising this as a comment because it's not a new problem that you're introducing here, but in general I'd be much happier if you could err on the side of cleaning up that sort of thing when you're touching code.

(and yes this is a systemic problem throughout react-sdk/riot. We can only hope to improve it incrementally.)

Reply:
That's fair, happy to leave a few more comments.